src/snowflake/cli/_plugins/sql/snowsql_commands.py

```diff
@@ -70,8 +71,22 @@ def _execute_help(self):
         filters.append(["status", "string", "any"])
         filters.append(["warehouse", "string", "any"])
         filters.append(["user", "string", "any"])
-        filters.append(["start", "date in milliseconds", "any"])
-        filters.append(["end", "date in milliseconds", "any"])
+        filters.append(
```

Comment:
Why we build this list instead of defining `filter =[ ... ]`? Defining the full list would be more efficient. WDYT?